src/utils/pause/pausePoints.js

```diff
@@ -22,8 +22,8 @@ function insertStrtAt(string, index, newString) {
 export function convertToList(pausePoints: PausePoints): PausePoint[] {
   const list = [];
   for (const line in pausePoints) {
-    for (const column in pausePoints[line]) {
-      const point = pausePoints[line][column];
+    for (const column in pausePoints[Number(line)]) {
+      const point = pausePoints[Number(line)][Number(column)];
```

Comment:
I think we can simplify this a lot if we have a location field in pause points from the beginning:

```diff
diff --git a/src/workers/parser/pausePoints.js b/src/workers/parser/pausePoints.js
index fa12cd70b..ef6bc40be 100644
--- a/src/workers/parser/pausePoints.js
+++ b/src/workers/parser/pausePoints.js
@@ -170,17 +170,19 @@ function hasPoint(state, { line, column }) {

 function addPoint(
   state,
-  { line, column },
+  location,
   types: boolean | { break?: boolean, step?: boolean }
 ) {
   if (typeof types === "boolean") {
     types = { step: types, break: types };
   }

+  const { line, column } = location;
+
   if (!state[line]) {
     state[line] = {};
   }
-  state[line][column] = types;
+  state[line][column] = { types, location };
   return state;
 }
```

I added location in my map pause points PR, but I don't think i was thinking [big] enough. I really should have done it in the worker so we always have it.

[big]: https://github.com/devtools-html/debugger.html/pull/7349/files#diff-44b80fbbfa2e03e79842d1e4b853ff20R42